packages/components/src/Banner/Banner.tsx

```diff
@@ -65,14 +65,14 @@ export function Banner({
     primaryAction = Object.assign(
       {
         size: "small",
-        type: "tertiary",
-        variation: type === "notice" ? "learning" : "work",
+        type: "secondary",
+        variation: type === "success" ? "work" : "learning",
       },
       primaryAction,
     );
   }
 
-  const bannerClassNames = classnames(styles.banner, {
+  const bannerClassNames = classnames(styles.banner, [styles[type]], {
```

Comment:
Took me way too long to get this working

Reply:
👏🏻